tests/test_implementations/test_rustsim_implementation.py

```diff
@@ -25,24 +30,27 @@ def setUp(self) -> None:
         if os.name == "nt":
             _, db = os.path.splitdrive(db)
 
-        oi = get_adapter(f"rustsim:sqlite:///{db}")
-
-        self.oi = oi
+        self.oi = get_adapter(f"rustsim:sqlite:///{db}")
         self.information_content_scores = {
             "CARO:0000000": 21.05,
             "BFO:0000002": 0.7069,
             "BFO:0000003": 14.89,
         }
         self.compliance_tester = ComplianceTester(self)
+        self.predicates = [IS_A, PART_OF]
+        self._rust_closure_table = relationships_to_closure_table(
```

Comment:
`_` indicates private method. No need to do this. The implementation should populate the closure table from the wrapped object. See the original oakx-grape code for an example